src/Templates/ProjectTemplates/CSharp/.NETCore/ConsoleApplication/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace $safeprojectname$
+{
+    internal class Program
+    {
+        internal static void Main(string[] args)
+        {
+        }
```

Comment:
Do we want to include the `Console.WriteLine("Hello World!");` line that is in the current 'dotnet new' template?